Paws4Me/Paws4Me/Controllers/SignInViewController.swift

```diff
@@ -12,47 +12,37 @@ class SignInViewController: UIViewController {
     @IBOutlet weak var txtPassword: UITextField!
     @IBOutlet weak var btnSignIn: UIButton!
     @IBOutlet weak var btnRegister: UIButton!
-    
     var bLoggedIn = false
-    
     override func viewDidLoad() {
         super.viewDidLoad()
-        
         btnSignIn.layer.cornerRadius = 10
         btnRegister.layer.borderWidth = 1.0
         btnRegister.layer.cornerRadius = 10
-        btnRegister.layer.borderColor = UIColor.init(red: 55/255,green: 21/255,blue: 67/255,alpha: 1).cgColor
-        
+        btnRegister.layer.borderColor = UIColor.init(red: 55/255, green: 21/255, blue: 67/255, alpha: 1).cgColor
         let bottomLine = CALayer()
-        
         let bottomLine2 = CALayer()
-        
-        bottomLine.frame = CGRect(x: 0, y: txtUsername.frame.height - 2, width: txtUsername.frame.width , height: 2)
-        
-        bottomLine2.frame = CGRect(x: 0, y: txtPassword.frame.height - 2, width: txtPassword.frame.width , height: 2)
-        bottomLine.backgroundColor = UIColor.init(red: 55/255,green: 21/255,blue: 67/255,alpha: 1).cgColor
-        
-        bottomLine2.backgroundColor = UIColor.init(red: 55/255,green: 21/255,blue: 67/255,alpha: 1).cgColor
-        
+        bottomLine.frame = CGRect(x: 0, y: txtUsername.frame.height*0.8, width: txtUsername.frame.width, height: 2)
+        bottomLine2.frame = CGRect(x: 0, y: txtPassword.frame.height*0.8, width: txtPassword.frame.width, height: 2)
+        bottomLine.backgroundColor = UIColor.init(red: 55/255, green: 21/255, blue: 67/255, alpha: 1).cgColor
```

Comment:
So all style kit comments and clean up will also be relevant in this PR.

Reply:
Thank you for the suggestion. 👍🏻